### PR TITLE
Updating helm deployment chart 

### PR DESCRIPTION
**Pull request by lieberlois**

### CODE REVIEW

File: deploy/helm/opsbot/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ replicaCount: 1
 
 image:
   repository: maibornwolff/opsbot
-  tag: v0.0.1
+  tag: dev
```
Comment (swoehrl-mw) [inline deploy/helm/opsbot/values.yaml]: tag must be `v0.0.1` for the release pipeline to work

File: deploy/helm/opsbot/templates/deployment.yaml [hunk 2]
```diff
@@ -48,9 +52,9 @@ spec:
             - name: plugins
               mountPath: /additional_plugins
             {{- end }}
-            {{- range .Values.extraVolumes }}
```
Comment (swoehrl-mw) [inline deploy/helm/opsbot/templates/deployment.yaml]: `extraVolumes` should be kept as an option

File: deploy/helm/opsbot/templates/ingress.yaml [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "opsbot.fullname" . -}}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: {{ $fullName }}
+  namespace: {{ .Values.ingress.namespace }}
```
Comment (swoehrl-mw) [inline deploy/helm/opsbot/templates/ingress.yaml]:
```suggestion
  namespace: {{ .Release.Namespace }}
```

File: deploy/helm/opsbot/templates/ingress.yaml [hunk 4]
```diff
@@ -14,26 +15,23 @@ metadata:
...
             backend:
-              serviceName: {{ $fullName }}
-              servicePort: http
-        {{- end }}
-  {{- end }}
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: {{ .Values.ingress.port | default 5000 }}
+      {{- if .Values.ingress.apiHost }}
```
Comment (swoehrl-mw) [inline deploy/helm/opsbot/templates/ingress.yaml]: Better name instead of `apiHost`: just `host`

File: deploy/helm/opsbot/templates/ingress.yaml [hunk 5]
```diff
@@ -14,26 +15,23 @@ metadata:
...
+            pathType: ImplementationSpecific
             backend:
-              serviceName: {{ $fullName }}
-              servicePort: http
-        {{- end }}
-  {{- end }}
+              service:
+                name: {{ $fullName }}
+                port:
+                  number: {{ .Values.ingress.port | default 5000 }}
```
Comment (swoehrl-mw) [inline deploy/helm/opsbot/templates/ingress.yaml]: port should not be configurable as it is hardcoded in the deployment and service

File: README.md [hunk 6]
```diff
@@ -91,6 +91,26 @@ persistence: # The persistence plugin to use. Currently available: file | config
...
+  enabled: false
+  namespace: opsbot # The namespace where the opsbot service is deployed
+  annotations:
+    # Example annotations
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/rewrite-target: /$1
+    nginx.ingress.kubernetes.io/use-regex: "true"
+  path: "/opsbot(/.*)"
+  port: 5000
+
```
Comment (swoehrl-mw) [inline README.md:112]: Add `host` as configurable parameter

File: README.md [hunk 7]
```diff
@@ -91,6 +91,26 @@ persistence: # The persistence plugin to use. Currently available: file | config
...
+  enabled: true
+
+pvc:
+  # Required when using the File plugin for persistence
+  enabled: true
+  size: "2Gi"
+
+ingress:
+  enabled: false
+  namespace: opsbot # The namespace where the opsbot service is deployed
```
Comment (swoehrl-mw) [inline README.md]: Not needed. Namespace is release namespace

**Review by swoehrl-mw — CHANGES_REQUESTED** [review]

File: deploy/helm/opsbot/values.yaml (hunk 1 above)
Comment (lieberlois) [reply deploy/helm/opsbot/values.yaml]: Done!

File: deploy/helm/opsbot/templates/deployment.yaml (hunk 2 above)
Comment (lieberlois) [reply deploy/helm/opsbot/templates/deployment.yaml]: Reverted changes to extraVolumes 👍

File: deploy/helm/opsbot/templates/ingress.yaml (hunk 3 above)
Comment (lieberlois) [reply deploy/helm/opsbot/templates/ingress.yaml]: Now using the release namespace 👍

File: deploy/helm/opsbot/templates/ingress.yaml (hunk 4 above)
Comment (lieberlois) [reply deploy/helm/opsbot/templates/ingress.yaml]: Renamed to host

File: deploy/helm/opsbot/templates/ingress.yaml (hunk 5 above)
Comment (lieberlois) [reply deploy/helm/opsbot/templates/ingress.yaml]: Hardcoded port in Service and Nginx Controller 👍

File: README.md (hunk 6 above)
Comment (lieberlois) [reply README.md:112]: Done

**Review by swoehrl-mw — APPROVED** [review]

**Pull request merged**